x/vaults/types/params.go

```diff
@@ -57,18 +52,12 @@ func (m Params) Validate() error {
 	if err := validateRecalculateDebtPeriod(m.ChargingPeriod); err != nil {
 		return err
 	}
-	if err := validateLiquidatePeriod(m.LiquidatePeriod); err != nil {
-		return err
-	}
 	return nil
 }
 
 func validateRecalculateDebtPeriod(i interface{}) error {
 	return nil
 }
-func validateLiquidatePeriod(i interface{}) error {
-	return nil
-}
 
 func validateStabilityFee(i interface{}) error {
```

Comment:
should move these validate to `ActiveCollateralProposal` and `UpdateCollateralProposal`